packages/wrangler/src/index.ts

```diff
@@ -129,6 +135,8 @@ After installation, run Wrangler with \`npx wrangler\`.`
 			);
 		}
 	}
+
+	return maybeNewVersion;
```

Comment:
Can we make updateCheck cache it's result (memoise the promise returned) so we don't need to return this here?

We can then call updateCheck again wherever we need, and avoid the props drilling through all the Dev components just to pass it to `new MiniflareServer`